src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] COLORS = {"white", "black", "red", "blue", "green", "yellow"};
+
+    public static String getRandomColor() {
+        Random random = new Random();

```

Comment:
we should create random instance once, not every time when we call this method
method should not be `static`. remove `static` and make related changes in the project
```suggestion
        private final Random random = new Random();

    public String getRandomColor() {
```